src/Acuminator/Acuminator.Utilities/BannedApi/Data/BannedApis.txt

```diff
@@ -0,0 +1,16 @@
+N:System.Data.Entity ISV Namespace usage is forbidden in Acumatica customizations.
```

Comment:
@EAndrosova please check the ban reasons that will be displayed by the diagnostic.